edgelet/edgelet-config/src/lib.rs

```diff
@@ -714,6 +736,20 @@ mod tests {
         };
     }
 
+    #[test]
+    fn external_prov_get_settings() {
+        let settings = Settings::<DockerConfig>::new(Some(Path::new(GOOD_SETTINGS_EXTERNAL)));
+        println!("{:?}", settings);
+        assert!(settings.is_ok());
```

Comment:
`Result::unwrap` already asserts that the `Result` is an `Ok`, so you don't need to assert it separately.

Reply:
Right. I also wanted to ensure that the config is actually read into the instance and not ignored by any chance. No harm in keeping the extra check imo.